src/controllers/roomController.js

```diff
@@ -54,6 +54,7 @@ class RoomController {
       }, { where: { id: roomId } });
       return successResponse(res, 200, 'Room updated');
     } catch (error) {
+      console.log(error);
```

Comment:
@funmi5 Please remove the `console.log()` statements in this file

Reply:
Oh yea I will, I was trying to resolve a failing test for rooms.